libraries/Update/examples/HTTPS_OTA_Update/HTTPS_OTA_Update.ino

```diff
@@ -0,0 +1,92 @@
+#include "WiFi.h"
+#include "HttpsOTAUpdate.h"
+
+static const char* ssid     = "your-ssid";  // your network SSID (name of wifi network)
+static const char* password = "your-password"; // your network password
+
+static const char * url = ""; //state url of your firmware image
```

Comment:
Please maintain consistency in the syntax and make changes as per ESP32 Arduino guidelines